src/index.html

```diff
@@ -5,8 +5,346 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <label for="MOYO">
+        <a href="index.html" class="header__cry">
+          <img
+            class="header__icon"
+            src="images/logo.png"
+            alt="MOYO"
+            id="MOYO"
+          >
+        </a>
+      </label>
+      <nav class="header__nav">
+        <ul class="header__menu">
+          <li class="header__list">
+            <a
+               class="header__link header__link--is-active is-active"
+               href="#">Apple</a>
+          </li>
+          <li class="header__list">
+            <a class="header__link" href="#">Samsung</a>
+          </li>
```

Comment:
```suggestion
      <label for="MOYO">
        <a href="index.html" class="header__cry">
          <img
            class="header__icon"
            src="images/logo.png"
            alt="MOYO"
            id="MOYO"
          >
        </a>
      </label>
      
      <nav class="header__nav">
        <ul class="header__menu">
          <li class="header__list">
            <a
               class="header__link header__link--is-active is-active"
               href="#">Apple</a>
          </li>
          
          <li class="header__list">
            <a class="header__link" href="#">Samsung</a>
          </li>
```
fix code style all over page